draft-ietf-quic-multipath.md

```diff
@@ -1337,12 +1325,15 @@ If a path is not actively used for a while, it might not be usable anymore,
 e.g. due to middlebox timeouts. To avoid such path breakage, endpoints
 can send ack-eliciting packets such as packets containing PING frames
 ({{Section 19.2 of QUIC-TRANSPORT}}) on that path to keep it alive.
-As discussed in
-{{Section 10.1.2 of QUIC-TRANSPORT}}, the keep-alive interval depends
-on the timeout in the middlebox.
-
-If a path was not actively used for a while, an endpoint can
-probe it before switching to active use if there are still other paths
+{{Section 5.3 of QUIC-TRANSPORT}} defines an optional keep alive process.
+This process can be applied to each path separately depending on application needs.
+Some applications may decide to not keep any not-actively used path alive,
+keep only one additonal path alive, or multiple paths, e.g. for more redunancy.
+As discussed in {{Section 10.1.2 of QUIC-TRANSPORT}}, the keep-alive interval
+needs to incorperate timeouts in middleboxes on the path.
+
+If a path was not actively used for a while and no keep alives have been sent,
```

Comment:
```suggestion
If a path was not actively used for a while and no more packets have been sent,
```

Reply:
I don't think this change puts it right. Active use effectively already implies that no packets have been sent, however, this second part is here to be explicit that also no keep alives have been sent because that is the kind of probing we talk about. If we want to change this sentence we could just remove the second part about keep alives but I think it's okay to be explicit.

Reply:
I think I agree with @mirjak.

Reply:
Okay, I will merge the PR now as it is.